packages/persisted-query-lists/README.md

```diff
@@ -1,3 +1,118 @@
 # `@apollo/persisted-query-lists`
 
```

Comment:
some sort of top-level description of the package?

i think it's worth saying basically, the link is for validating that you did the extraction properly and completely before moving on to actually send IDs; the other two functions let you send IDs, and there's a tradeoff between doing more work at runtime vs needing to get the just-in-time-generated manifest properly into the client build.

Reply:
Added an intro. Let me know what you think.